server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/java/org/apache/james/jmap/memory/MemoryDeletedMessagesVaultTest.java

```diff
@@ -47,4 +49,10 @@ protected GuiceJamesServer createJmapServer(FileSystem fileSystem, Clock clock)
     protected void awaitSearchUpToDate() {
 
     }
+
+    @Ignore("This feature is not supported")
```

Comment:
I'd say `This side effect behaviour is specific to Blobstore based implementation relying on deduplication, which is not the case of the tested implementation`.

Could you update the @Ignore message here and everywhere?